metldata/artifacts_rest/load_resources.py

```diff
@@ -101,3 +116,40 @@ async def load_artifact_resources(
             artifact_name=artifact_info.name,
             dao_collection=dao_collection,
         )
+
+
+async def process_removed_resources(
+    resource_tags: set[str], dao_collection: ArtifactDaoCollection
+):
+    """Delete no longer needed artifact resources from DB and send corresponding events"""
+
+    for resource_tag in resource_tags:
+        artifact_name, class_name, resource_id = resource_tag.split("#")
+        # resource tag was obtained from querying the db, so resource with given ID
+        # should be present
+        await remove_artifact_resource(
+            artifact_name=artifact_name,
+            class_name=class_name,
+            resource_id=resource_id,
+            dao_collection=dao_collection,
+        )
+
+    # needs event publisher and corresponding outgoing models here
```

Comment:
This comment is for GSI-278, I take it?

Reply:
Yep, just wanted placeholders for where to tentatively put stuff when moving code snippets around the code base.